pandas/_libs/tslib.pyx

```diff
@@ -54,6 +54,7 @@ from tslibs.nattype cimport checknull_with_nat, NPY_NAT
 from tslibs.timestamps cimport (create_timestamp_from_ts,
                                 _NS_UPPER_BOUND, _NS_LOWER_BOUND)
 from tslibs.timestamps import Timestamp
+from tslibs.offsets import localize_pydatetime  # noqa:F841
```

Comment:
would be ok with not doing this import and changing the tests

Reply:
After the current set of PRs I'm going to pitch an idea to 1) finish off `tslib` and 2) expose selected functions/classes in `tslibs.__init__`.  I'd rather avoid futzing with imports until then.

Reply:
ok that's fine